dimod/binary/binary_quadratic_model.py

```diff
@@ -601,18 +612,20 @@ def add_linear_equality_constraint(
         to the binary quadratic model as a quadratic objective.
 
         Args:
-            terms (iterable/iterator):
-                An iterable of 2-tuples, (variable, bias), with each tuple
-                constituting a term in :math:`\sum_{i} a_{i} x_{i},
-                with :math:`i` being the length of the iterable.
+            terms:
+                Values of the :math:`\sum_{i} a_{i} x_{i}` term as an
+                :math:`i`--length iterable of 2-tuples, ``(variable, bias)``, with
+                each tuple constituting a term in the summation.
             lagrange_multiplier:
-                A weight or the penalty strength. This value is
-                multiplied by the entire constraint objective and added to the
-                binary quadratic model (it does not appear explicitly in the
-                equation above).
+                Weight or penalty strength. The linear constraint is multiplied
+                by this value (which does not appear explicitly in the above
+                equation) when added to the binary quadratic model.
             constant:
-                The constant value of the constraint, :math:`C`, in the equation
-                above.
+                Value of the constant term, :math:`C`, of the linear constraint.
+
+        Examples:
+            >>> bqm = dimod.BinaryQuadraticModel("BINARY")
+            >>> bqm.add_linear_equality_constraint([("x1", 5), ("x2", -2)], 10, -3)
```

Comment:
this is deterministic, so I think you can safely show the output. Maybe with `.to_polystring()`?

Reply:
Did you mean to add the `.to_polystring()` method to BQMs too? It's now just on QMs.
I hadn't printed the BQM to keep the example short, but I can just print it the standard way.